doc/extensions/zephyr/link-roles.py

```diff
@@ -0,0 +1,40 @@
+# based on http://protips.readthedocs.io/link-roles.html
+
+from __future__ import print_function
+from __future__ import unicode_literals
+import re
+import os
+from docutils import nodes
+from local_util import run_cmd_get_output
+
+
+def get_github_rev():
+    tag = run_cmd_get_output('git describe --exact-match')
```

Comment:
When on an untagged branch (such as master) this writes out something like: ``fatal: no tag exactly matches '9d36fe0e2c7b4fdcb998d5ded5443ecffbcb19c9'`` I suspect to stderr so it shows up on the output of the ``make htmldocs``.  I think you can hide that stderr output ...